packages/seojaewan-react/src/hooks/useLogList/useLogList.test.ts

```diff
@@ -0,0 +1,41 @@
+import { act, renderHook } from "@testing-library/react";
+import useLogList from ".";
+
+const MESSAGE = "Test Log";
+const MESSAGE_LIST = ["Test Log1", "Test Log 2"];
+
+describe("useLogList 테스트", () => {
+  it("초기 logList 값인 [](빈 배열)이 반환된다.", () => {
+    const { result } = renderHook(() => useLogList());
+
+    expect(result.current.logList).toEqual([]);
+  });
+
+  it("addLog()에 전달된 'Test Log'가 logList에 추가된다.", () => {
+    const { result } = renderHook(() => useLogList());
+
+    act(() => result.current.addLog(MESSAGE));
+    expect(result.current.logList[0].message).toEqual(MESSAGE);
```

Comment:
아마도 의도하신 것은 아래와 같이 표현하는 편이 더 가까울 것 같습니다. `expect.objectCotnaing`을 사용하면 객체의 일부 속성만 비교할 수 있습니다.
```suggestion
    expect(result.current.logList).toEqual([
      expect.objectContaining({ message: MESSAGE })
    ]);
```

Reply:
오타가 있어서 수정했습니다 (머쓱...)

Reply:
제가 몰랐던 편의성 테스트 함수가 많이 있었네요...! 
테스트를 작성할  때, 기존 함수들을 좀 더 찾아보겠습니다!